docs/README.md

```diff
@@ -1,4 +1,4 @@
-# Kubernetes Documentation: HEAD
+# Kubernetes Documentation: releases.k8s.io/HEAD
```

Comment:
I meant for text - it is basically a link back to this same document - I
want it to be something people read, not click.

On Fri, Jul 10, 2015 at 9:18 AM, Zach Loafman <notifications@github.com>
wrote:

> In docs/README.md
> <https://github.com/GoogleCloudPlatform/kubernetes/pull/11062#discussion_r34370305>
> :
>
> > @@ -1,4 +1,4 @@
> > -# Kubernetes Documentation: HEAD
> > +# Kubernetes Documentation: releases.k8s.io/HEAD
>
> Did you mean this to be linky? 'Cuz it's raw text right now, which looks a
> little odd.
>
> —
> Reply to this email directly or view it on GitHub
> <https://github.com/GoogleCloudPlatform/kubernetes/pull/11062/files#r34370305>
> .
>